File: Chapter01/handling_data_gopher_style/example1/myprogram.py
import pandas as pd

# Define column names.
cols = [
        'integercolumn',
        'stringcolumn'
        ]

# Read in the CSV with pandas.
data = pd.read_csv('myfile.csv', names=cols)

# Print out the maximum value in the integer column.
print(data['integercolumn'].max())

File: Chapter01/handling_data_gopher_style/example2/myprogram.go
package main

import (
	"encoding/csv"
	"fmt"
	"log"
	"os"
	"strconv"
)

func main() {

	// Open the CSV.
	f, err := os.Open("myfile.csv")
	if err != nil {
		log.Fatal(err)
	}

	// Read in the CSV records.
	r := csv.NewReader(f)
	records, err := r.ReadAll()
	if err != nil {
		log.Fatal(err)
	}

	// Get the maximum value in the integer column.
	var intMax int
	for _, record := range records {

		// Parse the integer value.
		intVal, err := strconv.Atoi(record[0])
		if err != nil {
			log.Fatal(err)
		}

		// Replace the maximum value if appropriate.
		if intVal > intMax {
			intMax = intVal
		}
	}

	// Print the maxium value.
	fmt.Println(intMax)
}
